conf/ocsci/independent_rhcs_cluster.yaml

```diff
@@ -0,0 +1,12 @@
+DEPLOYMENT:
+  independent_mode: True
```

Comment:
@shylesh same.. wouldn't it be better to rename Independent to "External" everywhere

Reply:
done